src/Symfony/Component/String/ByteString.php

```diff
@@ -368,7 +368,12 @@ public function startsWith($prefix): bool
     public function title(bool $allWords = false): parent
     {
         $str = clone $this;
-        $str->string = $allWords ? ucwords($str->string) : ucfirst($str->string);
+
+        $limit = $allWords ? -1 : 1;
+
+        $str->string = preg_replace_callback('/\b./u', static function ($m): string {
```

Comment:
The `u` modifier must be removed `'/\b./'` - which means your test case should be moved to `AbstractUnicodeTestCase`. I'm not sure we can implement this change on `ByteString`: a high-ASCII byte might be a letter actually. It all depends on how `\b` figures out the word boundary.

Reply:
You are right, removed these code from `ByteString`. And moved test cases to `AbstractUnicodeTestCase`.